format/test/ls-files_test.sh

```diff
@@ -51,7 +50,11 @@ js=$(ls-files JavaScript src.js gen1.js gen2.js gen3.js)
     exit 1
 }
 js=$(ls-files JavaScript src.js gen1.js gen2.js gen3.js --disable_git_attribute_checks)
-[[ "$js" == "src.js gen1.js gen2.js gen3.js" ]] || {
+expected='src.js
+gen1.js
+gen2.js
+gen3.js'
+[[ "$js" == "$expected" ]] || {
```

Comment:
note that lower down in this file there is (existing) code in this structure, and expecting the return value to be spread over several liens